Web socket contestant list unit test failing

Django server unit tests failed with the following error:

```
======================================================================
FAIL: test_create_contestant_invalid_room (quiz.test_websocket.WebSocketContestantListTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/c/Users/Paul Crooks/Documents/Projects/qwiz/quiz/test_websocket.py", line 131, in test_create_contestant_invalid_room
    'John Doe'
AssertionError: {'contestants': ['John Doe', 'Jane Doe'], 'command': 'contestant_list'} != {'contestants': ['Jane Doe', 'John Doe'], 'command': 'contestant_list'}
- {'command': 'contestant_list', 'contestants': ['John Doe', 'Jane Doe']}
?                                                     ------------

+ {'command': 'contestant_list', 'contestants': ['Jane Doe', 'John Doe']}
?                                                ++++++++++++
```

This test failed because the dictionary returned was in a different order than expected. By definition, a dictionary is un-ordered so we shouldn't rely on this.

--- a/quiz/models.py
+++ b/quiz/models.py
@@ -149,7 +149,7 @@ class Room(models.Model):
         """
         message = {
             "command": "contestant_list",
-            "contestants":  [contestant.handle for contestant in self.contestant_set.all()]
+            "contestants":  sorted([contestant.handle for contestant in self.contestant_set.all()])
         }
         send_message(self.group_name(), message)
 

--- a/quiz/test_websocket.py
+++ b/quiz/test_websocket.py
@@ -127,8 +127,7 @@ class WebSocketContestantListTests(ChannelTestCase):
         self.assertDictEqual(response, {
             'command': 'contestant_list',
             'contestants': [
-                'Jane'
-                ' Doe',
+                'Jane Doe',
                 'John Doe'
             ]
         })

